configure.py

```diff
@@ -1344,7 +1347,13 @@ def write_config(data, name):
     if pkgicu[0] is None:
       error('''Could not load pkg-config data for "icu-i18n".
        See above errors or the README.md.''')
-    (libs, cflags, libpath) = pkgicu
+    (libs, cflags, libpath, icuversion) = pkgicu
+    o['variables']['icu_system_version'] = icuversion
+    icu_ver_major = icuversion.split('.')[0]
+    o['variables']['icu_ver_major'] = icu_ver_major
+    if int(icu_ver_major) < icu_versions["minimum_icu"]:
```

Comment:
`icuversion` could be x.y.z.  `minimum_icu` is an int, so this comparison is good.

If we ever need a certain dot release (60.2), we'll need to change the other path.

Also note that ICU version numbers start at `.1` for the GA release.